controllers/authController.js

```diff
@@ -0,0 +1,72 @@
+const User = require('../models/User')
+const bcrypt = require('bcryptjs');
+const {validationResult} = require('express-validator')
+const jwt = require('jsonwebtoken')
+const {secret} = require('../config/config')
+
+const generateAccessToken = (id) => {
+	const payload = {id}
+	return jwt.sign(payload, secret, {expiresIn: '24h'})
+}
+
+class authController {
+	async registration(req, res) {
+		try {
+			//const validationErrors = validationResult(req)
+			if (!validationResult(req).isEmpty()) return res.status(400).json({message: `Registration errors ${validationErrors}`})
+
+			const {email, firstName, lastName, phoneNumber, gender, birthDate, password} = req.body
+			const candidate = await User.findOne({ email })
+
+			if (candidate) {
+				res.status(400).json({message: 'This email is already used!'})
+			}
+			var hashPassword = bcrypt.hashSync(password, 9);
+			const user = new User({ 
+				email: email, 
+				firstName: firstName,
+				lastName: lastName, 
+				phoneNumber: phoneNumber, 
+				gender: gender, 
+				birthDate: birthDate, 
+				password: hashPassword 
+			})
```

Comment:
You can use [shorthand ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions)in these cases
```suggestion
			const user = new User({ 
				email, 
				firstName,
				lastName, 
				phoneNumber, 
				gender, 
				birthDate, 
				password: hashPassword,
			})
```